src/asset-swapper/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -46,6 +47,7 @@ import {
     getMaxQuoteSlippageRate,
     getTransformerNonces,
     isDirectSwapCompatible,
+    isFeeTransferTokenCompitable as isFeeTransferTokenFillCompatible,
```

Comment:
Maybe we can import this with the same name

Reply:
yes correct